atomic_reactor/utils/resource_manager.py

```diff
@@ -76,6 +77,27 @@ def slots(self):
     def slots(self, num: int):
         self._slots = num
 
+    @property
+    def slots_dir(self):
+        if self._slots_dir is None:
+            with self._ssh_session() as client:
+                home, _, _ = self._ssh_run_remote_cmd(client, "pwd")
+            self._slots_dir = os.path.join(home, SLOTS_RELATIVE_PATH)
```

Comment:
Huh, so the `resource_manager/` directory is in the current user's $HOME? That seems like it might cause issues. Would it be possible to use some system location?

How does the directory get created? Does the remote-hosts playbook do that?

Reply:
> Huh, so the `resource_manager/` directory is in the current user's $HOME? That seems like it might cause issues. Would it be possible to use some system location?

Yes, there could be some issues especially while using network-based accounts, e.g. ldap users but home is not created or mounted.

> How does the directory get created? Does the remote-hosts playbook do that?

We're current creating local users with default home directory created automatically in ansible role, so I'd leave it as it is now and find a more reliable solution when we need to support non-local users.

Reply:
Seems reasonable. What about the `resource_manager/` directory itself? It looks like the code expects this directory to already exist, does something create it?

Reply:
oops, you're right, good catch. what do you think about creating these slot files as `.osbs_slot_{num}.lock` in home directory and don't need to worry about creating the placeholder dir?

Reply:
Hmm, I liked having a dedicated directory for them, but it's not super important to me. Maybe they don't need to be hidden files? It sort of "feels right" to keep them visible, can't quite explain why though...